<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogNewBuild</class>
 <widget class="QDialog" name="dialogNewBuild">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Build</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/resources/logo.ico</normaloff>:/resources/logo.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="lineBuildName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPlatform">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>44</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>Concept X1000R (636 x 406 mm)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Concept X2000R (800 x 400 mm)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EOS M280/M290 (250 x 250 mm)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelBuildPlatform">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Build Platform</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBrowseBF">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>182</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Change where the captured images for the current build will be stored.</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBuildName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Build Name</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>395</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCalibrationFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>78</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Calibration File</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSliceFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>112</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Slice File(s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCalibrationFile">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>78</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBrowseCF">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>77</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBrowseSF">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>111</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBuildFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>182</y>
     <width>291</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listSliceFiles">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>112</y>
     <width>161</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelBuildFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>148</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Build Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>216</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUsername">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>216</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>52</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupNotifications">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>284</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Receive Notifications For:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>151</width>
      <height>130</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkAll">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkMinor">
       <property name="text">
        <string>Minor Defects</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkMajor">
       <property name="text">
        <string>Major Defects</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkFailure">
       <property name="text">
        <string>Capture Failure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkError">
       <property name="text">
        <string>Program Error</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="labelEmailAddress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Email Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEmailAddress">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>52</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSendTestEmail">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>284</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Test Email</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialogNewBuild</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialogNewBuild</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
